x-pack/legacy/plugins/infra/public/pages/logs/stream/page_logs_content.tsx

```diff
@@ -134,13 +134,8 @@ export const LogsPageLogsContent: React.FunctionComponent = () => {
                 <WithSummary>
                   {({ buckets }) => (
                     <WithLogPosition>
-                      {({
-                        isAutoReloading,
-                        jumpToTargetPosition,
-                        visibleMidpointTime,
-                        visibleTimeInterval,
-                      }) => (
-                        <WithStreamItems initializeOnMount={!isAutoReloading}>
+                      {({ jumpToTargetPosition, visibleMidpointTime, visibleTimeInterval }) => (
+                        <WithStreamItems initializeOnMount={false}>
```

Comment:
:information_source: This was the second initialization, which could lead to duplicate fetching.